File: b52-bomber/feature-tik/src/main/java/com/mobile/app/bomber/tik/mine/ContentDialogFragment.java
package com.mobile.app.bomber.tik.mine;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;

import com.mobile.guava.android.mvvm.BaseAppCompatDialogFragment;

import com.mobile.app.bomber.common.base.tool.SingleClick;
import com.mobile.app.bomber.tik.databinding.FragmentCommonDialogBinding;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class ContentDialogFragment extends BaseAppCompatDialogFragment implements View.OnClickListener {

    private FragmentCommonDialogBinding binding;
    private String content;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        binding = FragmentCommonDialogBinding.inflate(inflater, container, false);
        binding.confirm.setOnClickListener(this);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NotNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        content = getArguments().getString("content");
        binding.contentTv.setText(content);
    }

    @SingleClick
    @Override
    public void onClick(View v) {
        dismissAllowingStateLoss();
    }

    public static ContentDialogFragment newInstance(String content) {
        ContentDialogFragment fragment = new ContentDialogFragment();
        Bundle bundle = new Bundle();
        bundle.putString("content", content);
        fragment.setArguments(bundle);
        return fragment;
    }
}

File: b52-bomber/feature-tik/src/main/java/com/mobile/app/bomber/tik/mine/ContentDialogFragment.kt
package com.mobile.app.bomber.tik.mine

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.mobile.app.bomber.common.base.tool.SingleClick
import com.mobile.app.bomber.tik.databinding.FragmentCommonDialogBinding
import com.mobile.guava.android.mvvm.BaseAppCompatDialogFragment

class ContentDialogFragment : BaseAppCompatDialogFragment(), View.OnClickListener {
    private var _binding: FragmentCommonDialogBinding? = null
    private val binding get() = _binding!!
    private var content: String? = null
    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        dialog?.window?.setBackgroundDrawableResource(android.R.color.transparent)
        _binding = FragmentCommonDialogBinding.inflate(inflater, container, false)
        binding.confirm.setOnClickListener(this)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        content = arguments?.getString("content")
        binding.contentTv.text = content
    }

    @SingleClick
    override fun onClick(v: View) {
        dismissAllowingStateLoss()
    }

    companion object {
        @JvmStatic
        fun newInstance(content: String?): ContentDialogFragment {
            val fragment = ContentDialogFragment()
            val bundle = Bundle()
            bundle.putString("content", content)
            fragment.arguments = bundle
            return fragment
        }
    }
}